plotting-app/hid-driver/pom.xml

```diff
@@ -0,0 +1,27 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <parent>
+        <artifactId>plotting-app</artifactId>
+        <groupId>org.unknown.plotting</groupId>
```

Comment:
did we miss out to name package properly?

Reply:
Not really. That's the name I went with for now